src/scalarstats.jl

```diff
@@ -166,19 +222,51 @@ nquantile{T<:Real}(v::AbstractArray{T}, n::Integer) = quantile(v, (0:n)/n)
 #############################
 
 # span, i.e. the range minimum(x):maximum(x)
+"""
+    span(x::AbstractArray{T})
+
+Returns the range of `x`, i.e., minimum(x):maximum(x).
+"""
 span{T<:Integer}(x::AbstractArray{T}) = ((a, b) = extrema(x); a:b)
 
 # Variation coefficient: std / mean
 variation{T<:Real}(x::AbstractArray{T}, m::Real) = stdm(x, m) / m
 variation{T<:Real}(x::AbstractArray{T}) = variation(x, mean(x))
+"""
+    variation(x::AbstractArray{T}, m::Real)
+
+### Args:
+* `x`: An array of type `AbstractArray`.
+* `m`: An optional paramater, which specifies the mean.
+
+Computes the ratio of standard deviation to mean.
+"""
+function variation end
 
 # Standard error of the mean: std(a) / sqrt(len)
+"""
+    sem(a::AbstractArray{T})
+
+### Args:
+* `a`: An array of type `AbstractArray`.
+
+Computes the standard error of the mean, i.e. sqrt(var / n).
+"""
 sem{T<:Real}(a::AbstractArray{T}) = sqrt(var(a) / length(a))
 
 # Median absolute deviation
 mad{T<:Real}(v::AbstractArray{T}, args...;arg...) = mad!(copy(v), args...;arg...)
 mad{T<:Real}(v::Range{T}, args...;arg...) = mad!([v;], args...;arg...)
-
+"""
+    mad(x, center;constant=1.4826)
+### Args:
+* `x`: An array of type `AbstractArray`.
+* `center`: Optional parameter which specifies the center of `x`.
+* `constant`: Optional paramter, Constant scale factor, default is 1.4826.
```

Comment:
Typo here and in the same place above, but I'd rather remove these argument lists anyway.